src/java/org/apache/cassandra/index/sai/disk/vector/CassandraOnHeapGraph.java

```diff
@@ -552,7 +556,7 @@ private long writePQ(SequentialWriter writer, IntUnaryOperator reverseOrdinalMap
         return writer.position();
     }
 
-    private VectorCompressor<?> computeOrRefineFrom(CompressedVectors previousCV, VectorCompression preferredCompression)
+    VectorCompressor<?> computeOrRefineFrom(CompressedVectors previousCV, VectorCompression preferredCompression)
```

Comment:
Nit: is this intentional? It isn't used anywhere, AFAICT.

Reply:
was part of the next refactor that I left in by mistake, will put it back